planner/core/common_plans.go

```diff
@@ -384,6 +384,19 @@ func (e *Execute) getPhysicalPlan(ctx context.Context, sctx sessionctx.Context,
 	sessVars := sctx.GetSessionVars()
 	stmtCtx := sessVars.StmtCtx
 	prepared := preparedStmt.PreparedAst
+	// disable the cache if cache table in prepared statement
+	for _, vInfo := range preparedStmt.VisitInfos {
```

Comment:
Can we move it to line 414?   After the first time check, `prepared.UseCache` remain is false and not checked anymore?

If you put it here, the performance of every query is affected even you don't use the small cached table. @sylzd